components/dialog/index.css

```diff
@@ -34,337 +34,321 @@ governing permissions and limitations under the License.
   --spectrum-dialog-content-margin-bottom: var(--spectrum-global-dimension-size-450);
 
-  /* Bug: this must be 0ms, not 0 */
-  --spectrum-dialog-exit-animation-delay: 0ms;
+  /* 12px desktop (medium scale) and 8px mobile (large scale) */
+  --spectrum-dialog-close-button-padding: calc(26px - var(--spectrum-global-dimension-size-175));
+  --spectrum-dialog-close-button-size: var(--spectrum-global-dimension-size-400);
+
+  --spectrum-dialog-gap-size: var(--spectrum-global-dimension-size-200);
 }
 
-.spectrum-Dialog {
-  @inherit: %spectrum-overlay;
 
+.spectrum-Dialog {
   /* Be a flexbox to allow a full sized content area that scrolls */
   display: flex;
-  flex-direction: column;
-
-  /* Centered by default */
-  position: fixed;
-  left: 50%;
-  top: 50%;
-
-  /* Start offset by the animation distance */
-  transform: translateX(-50%) translateY(-50%) translateY(var(--spectrum-dialog-entry-animation-distance));
-
-  /* Appear above underlay */
-  z-index: 2;
 
   /* Allow 100% width, taking into account padding */
   box-sizing: border-box;
 
-  /* Don't be bigger than the screen */
-  max-inline-size: var(--spectrum-dialog-max-width);
-  max-block-size: var(--spectrum-dialog-max-height);
-
   /* Be no bigger than max-width, but also be 90% if the viewport is smaller than max-width */
-  inline-size: fit-content;
-  min-inline-size: var(--spectrum-dialog-min-width);
-
-  padding: var(--spectrum-dialog-padding);
+  width: fit-content;
+  min-width: var(--spectrum-dialog-min-width);
+  max-width: 100%;
 
-  border-radius: var(--spectrum-dialog-border-radius);
-  outline: none; /* Firefox shows outline */
+  max-height: inherit;
 
-  /* Exit animations */
-  transition: opacity var(--spectrum-dialog-exit-animation-duration) var(--spectrum-global-animation-ease-in) var(--spectrum-dialog-exit-animation-delay),
-              visibility 0ms linear calc(var(--spectrum-dialog-exit-animation-delay) + var(--spectrum-dialog-exit-animation-duration)),
-              transform 0ms linear calc(var(--spectrum-dialog-exit-animation-delay) + var(--spectrum-dialog-exit-animation-duration));
-
-  &.is-open {
-    @inherit: %spectrum-overlay--open;
-
-    /* Entry animations */
-    transition: transform var(--spectrum-dialog-entry-animation-duration) var(--spectrum-global-animation-ease-out) var(--spectrum-dialog-entry-animation-delay),
-                opacity var(--spectrum-dialog-entry-animation-duration) var(--spectrum-global-animation-ease-out) var(--spectrum-dialog-entry-animation-delay);
-
-    transform: translate(-50%, calc(-50% + var(--spectrum-dialog-vertical-offset)));
-  }
-
-  .spectrum-Dialog-closeButton {
-    display: none;
-  }
+  outline: none;
 }
 
 .spectrum-Dialog--small {
-  inline-size: var(--spectrum-dialog-small-width);
+  width: var(--spectrum-dialog-small-width);
 }
 
 .spectrum-Dialog--medium {
-  inline-size: var(--spectrum-dialog-medium-width);
+  width: var(--spectrum-dialog-medium-width);
```

Comment:
are we switching back to width instead of inline-size? I thought this was for RTL support